test/integration/targets/rabbitmq_lookup/aliases

```diff
@@ -0,0 +1,5 @@
+destructive
+shippable/posix/group1
+skip/osx
+skip/freebsd
+skip/rhel
```

Comment:
What is the problem with RHEL ?

Reply:
I had just copied the existing rabbitmq integration tests aliases file.  I've removed skip freebsd and rhel now.  CentOS 7 test appears to work fine.